### ISSUE TITLE
Regionalise enterprise sign up screens 

### ISSUE THREAD

**Issue by MagsHall** [openfoodfoundation/openfoodnetwork#1040]
Enterprise sign up process on UK production still contains a number of Aus references which don't make sense or look unprofessional in UK setting. 
![need to regionalise the sign up page](https://cloud.githubusercontent.com/assets/19837861/16076693/82ad7cce-32ec-11e6-9482-6c62a65a063b.png)
Change 'suburb' to 'Town / City'. Change 'State' to 'County' and if possible list the full county / region names rather than the first three letters - this isn't common in the UK and doesn't always make sense. 

![regionalise set up page 2](https://cloud.githubusercontent.com/assets/19837861/16076729/c8dac54e-32ec-11e6-9714-a64144c21291.png)
Remove ABN and ACN fields 

Not a huge priority but is a bit annoying

**Comment by lin-d-hop** [comment]
These are coming with the next deploy. Changed in transifex.

**Comment by lin-d-hop** [comment]
Lots has been changed now. Can you flag up anything else you find please mags?

**Comment by MagsHall** [comment]
Yup, looking good! I'd still be tempted to turn 'suburb' into 'town / city' as I'd never use the word suburb, but maybe that's a Scots thing.

**Comment by NickWeir63** [comment]
i agree that town/city would be better

**Comment by MagsHall** [comment]
When helping set up a new user just now we spotted that the 'State' drop down has reverted to the three letter abbreviations. Am I right in thinking it had been changed to a 'County' box with full county names? I can't quite remember if that was one of the things that had been fixed. 

The full county names are available in the Edit Enterprise Address tab in Spree still. Has something gone wrong with the last update?
